Gemfile

```diff
@@ -1,3 +1,5 @@
 source "https://rubygems.org"
 
+gem 'rspec', :require => 'spec'
```

Comment:
Hmm, yeah, I dunno either.  I hit a problem running spec tests but I can't seem to reproduce it anymore, so I'll just remove this commit.